compose/README.md

```diff
@@ -0,0 +1,20 @@
+This is a repo to hold all the bits to get a prod-like Normandy running using
+[Docker Compose][].
+
+1. Install [Docker][] and Docker Compose, and start the Docker daemon (possibly via [Docker Machine][]).
+2. Clone this repo, and `cd` into it.
```

Comment:
I haven't updated these instructions to account for the monorepo yet. I'll probably do that as a unified docs pass.